apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data.IntegrationTests/ReadTests.cs

```diff
@@ -372,10 +372,35 @@ private class CredentialWrapper : ITokenAccess
         }
 
         [Fact]
+        public async Task CommandTimeout()
+        {
+            long result = 0;
+            var e = await Assert.ThrowsAsync<SpannerException>(
+                async () =>
+                {
+                    using (await _testFixture.GetTestDatabaseConnectionAsync())
+                    {
+                        using (var connection =
+                            new SpannerConnection(_testFixture.ConnectionString))
+                        {
+                            var cmd =
+                                connection.CreateSelectCommand("SELECT 1");
+                            cmd.CommandTimeout = 0;
```

Comment:
The docs for DbCommand.CommandTimeout include:

> Note to implementers, it is recommended that 0 means no timeout.

Do we definitely want to violate that? (I understand it makes it harder to test otherwise, but...)